src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +96,22 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String star} into a {@code Star}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code Star} is invalid.
+     */
+    public static Star parseStar(String star) throws ParseException {
+        requireNonNull(star);
+        String trimmedStar = star.trim(); // trim the string to only get the number
+        Integer starCount = Integer.parseInt(trimmedStar);
```

Comment:
This will throw an uncaught NumberFormatException if `trimmedStar` is not a number. Maybe use `StringUtil.isNonZeroUnsignedInteger` to verify this first?